app/views/announcements/_ind_announcement_admin.html.erb

```diff
@@ -11,10 +21,13 @@
             <%= link_to 'View PDF', announcement.pdf.url %>
           <% end %>
         <% end %>
+      </div>        
+
+      <div class='text-subline'>
+        <div class='spaced-text'><%= announcement.description %>
```

Comment:
Stylistically it's better to leave the `<div>` tags on separate lines and put the content below, but #minor